app/controllers/api/alert_definitions_controller.rb

```diff
@@ -0,0 +1,4 @@
+module Api
+  class AlertDefinitionsController < BaseController
+  end
+end
```

Comment:
we need support code here, minimally for create.  I got a 500 internal server error creating an alert definition with bogus attributes. 

Need to test for minimally description, also validation of expression attribute so bogus ones aren't created.  see arbitration_rules_controller.rb or conditions_controller.rb.

create will need to be rescued returning a BadRequestError.